src/command-line/users/add.js

```diff
@@ -32,33 +32,48 @@ program
 		log.prompt({
 			text: "Enter password:",
 			silent: true,
-		}, function(err, password) {
+		}, (err, password) => {
 			if (!password) {
 				log.error("Password cannot be empty.");
 				return;
 			}
-			if (!err) {
+
+			if (err) {
+				return;
+			}
+
+			log.prompt({
+				text: "Save logs to disk?",
+				default: "yes",
+			}, (err2, enableLog) => {
```

Comment:
Funny, I actually tried to refactor to promises in this PR, but since we need all intermediary results in the last promise, we need to... nest the promises lol. It's actually common to nest promises when you need intermediary results, so people get frustrated because they think it kills the point of promises that are meant to remove callback hell, [which isn't the point](https://blog.domenic.me/youre-missing-the-point-of-promises/) 😁

So yeah, I ended up with nested promises, and I figured it wasn't worth the effort in this particular PR.

Reply:
`Promise.all` would solve your problem there. But sure, not in this PR.